docs/plugins/mbsubmit.rst

```diff
@@ -0,0 +1,54 @@
+MusicBrainz Submit Plugin
+=========================
+
+The ``mbsubmit`` plugin provides an extra prompt choice during an import
+session that prints the tracks of the current album in a format that is
+parseable by MusicBrainz's `track parser`_.
+
+.. _track parser: http://wiki.musicbrainz.org/History:How_To_Parse_Track_Listings
+
+Usage
+-----
+
+Enable the ``mbsubmit`` plugin in your configuration (see :ref:`using-plugins`)
+and select the ``Print tracks`` choice which is by default displayed when no
+strong recommendations are found for the album::
+
+    No matching release found for 3 tracks.
+    For help, see: http://beets.readthedocs.org/en/latest/faq.html#nomatch
+    [U]se as-is, as Tracks, Group albums, Skip, Enter search, enter Id, aBort,
+    Print tracks? p
+    01. An Obscure Track - An Obscure Artist (3:37)
+    02. Another Obscure Track - An Obscure Artist (2:05)
+    03. The Third Track - Another Obscure Artist (3:02)
+
+    No matching release found for 3 tracks.
+    For help, see: http://beets.readthedocs.org/en/latest/faq.html#nomatch
+    [U]se as-is, as Tracks, Group albums, Skip, Enter search, enter Id, aBort,
+    Print tracks?
+
+As MusicBrainz currently does not support submitting albums programmatically,
+the recommended workflow is to copy the output of the ``Print tracks`` choice
+and paste it into the parser that can be found by clicking on the
+"`Track Parser`" button on MusicBrainz "`Tracklist`" tab.
```

Comment:
Any particular reason for the extra backticks here? It seems like quotes might be enough. (This is officially the tiniest style issue ever!)